### PR TITLE
Additional NamespaceSelector info

### PR DESCRIPTION
**Pull request by tmjd**
Trying to add a little more information to the namespace selector and what it does and how to use it.

### CODE REVIEW

**Comment by emanic** [comment]
Thanks @tmjd !

File: _includes/master/selectors.md [hunk 1]
```diff
@@ -2,9 +2,12 @@ A label selector is an expression which either matches or does not match an endp
 
 {{site.prodname}} label selectors support a number of syntactic primitives.  Each of the following 
 primitive expressions can be combined using the logical operator `&&`.
+If any selector is used in a Namespaced context then it is limited to match
```
Comment (caseydavenport) [inline _includes/master/selectors.md]: I'd probably make this statement more concrete - for example:

"selectors on NetworkPolicy resources select only objects that are in the same namespace as the policy. Selectors on GlobalNetworkPolicy objects select across all namespaces"

Main thing is there is no definition of "in a namespaced context"

Comment (caseydavenport) [reply _includes/master/selectors.md]: Scratch that, I see what you did with the definition of namespace context above.

File: _includes/v3.0/selectors.md [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
 
-Calico label selectors support a number of syntactic primitives.  Each of the following 
+{{site.prodname}} label selectors support a number of syntactic primitives.  Each of the following 
 primitive expressions can be combined using the logical operator `&&`.
+If any selector is used in a Namespaced context then it is limited to match
+only the resources that are in that context.
 
 | Syntax              | Meaning                     |
 |---------------------|-----------------------------|
+| all()               | Match all endpoints.
```
Comment (caseydavenport) [inline _includes/v3.0/selectors.md]: Should probably replace endpoints with resources here as well.

File: _includes/master/entityrule.md [hunk 3]
```diff
@@ -7,3 +7,10 @@
 | namespaceSelector | Positive match on selected namespaces. If specified, only workload endpoints in the selected Kubernetes namespaces are matched. | Valid selector | [selector](#selector) | |
 | ports | Positive match on the specified ports | | list of [ports](#ports) | |
 | notPorts | Negative match on the specified ports | | list of [ports](#ports) | |
+
+##### Namespace Selector
```
Comment (caseydavenport) [inline _includes/master/entityrule.md]: A bit odd having this be the only thing outside of the table - could we just enhance the description in the table instead?

File: _includes/master/selectors.md [hunk 4]
```diff
@@ -2,9 +2,12 @@ A label selector is an expression which either matches or does not match an endp
```
Comment (caseydavenport) [inline _includes/master/selectors.md]: There are some changes in the v3.0 one that aren't in this file.

namely, "endpoint" -> "a resource"

File: _includes/master/selectors.md (hunk 1 above)
Comment (caseydavenport) [reply _includes/master/selectors.md]: Still feel like this sentence is confusing... but not sure how to make it more clear.

**Comment by tmjd** [comment]
Updated.
I decided to drop the additional comments in selector because it is not simple to describe what they apply to in the selector and instead added some additional wording in the entity rule for the selector and notSelector entries.  (It is extra tricky in the selector because those also apply to namespaces.)
PTAL

File: _includes/master/selectors.md (hunk 4 above)
Comment (caseydavenport) [reply _includes/master/selectors.md]: @tmjd just missing one diff in this file still

Comment (tmjd) [reply _includes/master/selectors.md]: Ok should be updated now. Sorry I copied over the whole table and didn't realize it was something in the text above it.

**Comment by tmjd** [comment]
@emanic Did you want to take a look now?

**Comment by emanic** [comment]
LGTM, thanks @tmjd

**Pull request merged**